test_book/src/individual/mixed.md

```diff
@@ -29,9 +29,16 @@ fn main(){
 
 <kbd>Ctrl</kbd> + <kbd>S</kbd> saves a file.
 
-A random image sprinkled in between
+---
+
+Although markdown does not have a native figure caption syntax, one can use block quote as a general-purpose "environment" to emulate it, and max-level heading inside the quote as caption:
+
```

Comment:
Can you say why this was modified? I think this test was just help with showing that images work at all and how styling works.  This seems unrelated to adding a test page for MathJax? Adding this seems to not be in the spirit of this test, which is to have simple direct examples of various markdown elements.